### DISCUSSION TITLE
Alarm clock trigger problem

### DISCUSSION THREAD

**Discussion by tuopu** [openthread/openthread#7467] [Q&A] [answered]
Hello, I'm trying to transplant openthread to the system. Here's a question for you. See the function void otplatalarmmillistartat (otinstance * ainstance, uint32_t at0, uint32_t ADT); When looking at the transplantation case, this function is not blocked. Please ask where the alarm clock triggered by this function works.

**Comment by jwhui** [comment, marked as answer]
When the alarm fires, it should call [`otPlatAlarmMilliFired()`](https://openthread.io/reference/group/plat-alarm#group__plat-alarm_1ga82ef729aca5865bf448dcaba69e46258).
